models/category/src/presets/index.ts

```diff
@@ -0,0 +1,3 @@
+export default {
+  // @TODO: add `category-draft`
```

Comment:
although it is nothing I would do yet,
it makes a lot more sense to add the draft models in the same package as the main model instead of having separate packages.

what do you think?
cc @ahmehri @tdeekens

Reply:
No hard opinion here, I mean I get it, it makes sense to group both the main and draft models together, which is better from a code colocation point of view. I don't see a reason why to not do it.